setup.cfg

```diff
@@ -0,0 +1,6 @@
+[flake8]
```

Comment:
The code style information is read from `tox.ini` and enforced by the test runner via `pycodestyle`. See `runtests.py`.